scylla/src/transport/cluster.rs

```diff
@@ -185,6 +185,7 @@ impl Cluster {
             &None,
             host_filter.as_deref(),
             TabletsInfo::new(),
+            &HashMap::new(),
         )
```

Comment:
:pick: typo in commit `Move handling of MissingUserDefinedType to ClusterData::new`: `to from` -> `from`